include/openthread/platform/logging.h

```diff
@@ -106,6 +106,14 @@ extern "C" {
  */
 #define OT_LOG_LEVEL_DEBG 5
 
+/**
+ * Maximum length for log file.
+ *
+ * @note This is only used when using OPENTHREAD_CONFIG_LOG_OUTPUT_FILE.
+ *
+ */
+#define OT_LOG_FILE_NAME_MAX_LEN 128
```

Comment:
This seems to be more platform implementation specific (I don't think OT core would need to be aware of file name size). How about removing it from `logging.h` and adding it under simul platform and `file_logging`?